index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link href="https://fonts.googleapis.com/css?family=Inter:400,600,500,800" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style/style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <!-- Welcome Header -->
+   <header class="header">
+       <h1>Welcome</h1>
+       <i class="fa fa-bars" aria-hidden="true"></i>
+   </header>
+
+   <!-- Main content -->
+    <section class="main1">
+        <div class="wrap-content">
+
+        <div class="intro">
+            <h5 class="name">Hey there. I'm Ntare Guy</h5>
+            <h5 class="prof">I am Software Developer</h5>
+        
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+            see and have a project you need coded,
+            don’t hestiate to contact me.</p>
+        </div>
+        <div class="social-media">
```

Comment:
[ ] [REQUIRED] Please enclose the list of icons with a `<ul>` tag to group them, and `<li>` per icon; then enclose each icon in an `<a>` tag because the icons will be used as links when clicked on.😊